containers/elm-0.19.0/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Elm (0.19.0)",
+	"dockerFile": "Dockerfile",
+
+	// Uncomment the next line if you want to publish any ports.
+	// 8000 is the default port used for the `elm reactor` command
+	//"appPort": [8000],
+
+	// Uncomment the next line if you want to add in default container specific settings.json values
+	// "settings":  { "workbench.colorTheme": "Quiet Light" },
+
+	// Uncomment the next line to run commands after the container is created.
+	// "postCreateCommand": "yarn install",
```

Comment:
Is yarn used with Elm?  If not, probably makes sense to change this example.

Reply:
Resolved in [2165d17c43](https://github.com/microsoft/vscode-dev-containers/pull/101/commits/2165d17c43709799143e24901583ea1c2f1bd26f)